packages/@tinacms/toolkit/src/packages/form-builder/FormBuilder.tsx

```diff
@@ -109,31 +109,31 @@ export const FormBuilder: FC<FormBuilderProps> = ({
   /**
    * Prevent navigation away from the window when the form is dirty
    */
-  // React.useEffect(() => {
-  //   const onBeforeUnload = (event) => {
-  //     event.preventDefault()
-  //     event.returnValue = ''
-  //   }
+  React.useEffect(() => {
+    // const onBeforeUnload = (event) => {
```

Comment:
@jeffsee55 I see you [commented this out recently](https://github.com/tinacms/tinacms/commit/47779b615c85cb5c83c72cd6e79b2a8b75664ee4#diff-e601155aaf8b0e6c16d2b919400b21642f76761aef56119d27ce33d1be4150fbR93)
This look okay to add back in?

Reply:
Yeah, commented out because of the alert, so just keeping this portion in seems good!